pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -65,6 +65,7 @@ func TestRemoveContainer(t *testing.T) {
 	fakeOS := m.osInterface.(*containertest.FakeOS)
 	fakeOS.GlobFn = func(pattern, path string) bool {
 		pattern = strings.Replace(pattern, "*", ".*", -1)
+		pattern = strings.Replace(pattern, "\\", "\\\\", -1)
```

Comment:
does glob as implemented by the OS require escaping on backslashes?

Reply:
No, ``RealOS.Glob`` simply calls ``filepath.Glob``, which works just fine:

```
package main

import (
    "fmt"
    "path/filepath"
)

func main() {
    path := "C:\\tmp\\*"

    globs, err := filepath.Glob(path)
    fmt.Printf("Globing %s: %+v, err: %v\n", path, globs, err)
}
```

Output:

```
go run glober.go
Globing C:\tmp\*: [C:\tmp\dat1 C:\tmp\dat1.yaml C:\tmp\dat2 C:\tmp\dat2.yaml C:\tmp\hostPath_pv.105f0794-79a8-4d68-a359-b12df3f6a61e C:\tmp\hostPath_pv.36f30545-21f0-4c1a-b1a2-1d6e33de0eaf], err: <nil>
```